.eslintrc

```diff
@@ -2,14 +2,11 @@
   "env": {
```

Comment:
We can probably use the eslint:recommended rules here.

Reply:
Which ones the recommended rules?

I normally use the air-bnb style.